iam_role.go

```diff
@@ -33,6 +33,19 @@ type IamRoleResponse struct {
 	RoleIPArn     string `json:"instanceProfileArn"`
 	RoleAddedToIP bool   `json:"addedRoleToInstanceProfile"`
 	Exists        bool   `json:"roleExists"`
+	AlksAccess    bool   `json:"machineIdentity"`
+}
+
+// GetIamRoleResponse is used to represent a a IAM Role.
+type GetIamRoleResponse struct {
```

Comment:
Any reason that this struct exists when it's a duplicate of the `IamRoleResponse` struct?  Maybe I missed the use case.

Reply:
Oops, looks like I forgot to delete `AlksAccess` from the `IamRoleResponse` struct, I made `GetIamRoleResponse` because only that endpoint returned the additional `machineIdentity` field, good catch